Dockerfile

```diff
@@ -50,33 +50,26 @@ RUN curl -L -o- https://download.docker.com/linux/static/stable/x86_64/docker-$D
     chmod 755 /usr/local/bin/docker
 
 ARG KUBE_VERSION=1.19.7
-RUN curl -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl && \
+RUN curl -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/$(dpkg --print-architecture)/kubectl && \
     chmod 755 /usr/local/bin/kubectl
 
 ARG HELM_VERSION=3.5.2
-RUN curl -L -o- https://get.helm.sh/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && \
-    cp linux-amd64/helm /usr/local/bin/helm && \
-    chmod 755 /usr/local/bin/helm
+RUN curl -fsSL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 -o get-helm-3.sh && \
+    chmod 700 get-helm-3.sh && \
+    ./get-helm-3.sh --version v${HELM_VERSION}
 
 ARG KOPS_VERSION=1.19.1
-RUN curl -Lo kops https://github.com/kubernetes/kops/releases/download/v$KOPS_VERSION/kops-linux-amd64 && \
+RUN curl -Lo kops https://github.com/kubernetes/kops/releases/download/v$KOPS_VERSION/kops-linux-$(dpkg --print-architecture) && \
     chmod +x ./kops && \
     mv ./kops /usr/local/bin/
 
 ARG ISTIO_VERSION=1.9.2
 RUN curl -L https://istio.io/downloadIstio | ISTIO_VERSION="$ISTIO_VERSION" sh - && \
     mv /istio-$ISTIO_VERSION/bin/istioctl /usr/local/bin/istioctl
 
-ARG K3D_VERSION=4.4.4
+ARG K3D_VERSION=4.4.8
 RUN curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v$K3D_VERSION bash
 
-# Install edgectl
-RUN curl -fL https://metriton.datawire.io/downloads/linux/edgectl -o /usr/local/bin/edgectl && \
-    chmod a+x /usr/local/bin/edgectl
```

Comment:
this is deprecated and also does not have `arm64` support 😞

Reply:
have not found a specific announcement for this, only this commit - https://github.com/emissary-ingress/emissary/commit/01b1f4eeabf1022b725198431044bf8455015e33